_posts/2021-01-14-voicemeeter-banana.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title:  "Introduction to Voicemeeter with a Banana"
+author: flavius
+categories: [ audio ]
+image: "voicemeeter.webp"
+featured: false
+---
+
+## What is Voicemeeter Banana
+
+Voicemeter Ba na na, na na na na, sing with me! Unless you are streaming or making a youtube video as that could result in a DMCA! Joke aside, Voicemeeter Banana, what is it? Voicemeeter Banana is the second iteration of Voicemeeter, an audio mixing software that allows advanced audio mixing, which is also provided free. You can download voicemeeter by visiting their website [here](http://voicemeeter.com/).
+
+Voicemeeter Banana comes with three Hardware Inputs, two software o digital inputs, and three hardware outputs. For reference, Hardwar inputs are typically hardware devices pushing audio into the computer, such as microphones, line inputs, and other things like that. The software or digital input is a driver on the system that allows audio to be pushed to it, they help with threading audio around. Hardware outputs are your headphones, speakers and can also be a digital audio driver.
+
+## Installing Voicemeeter Banana
+
+I think the best way to explain Voicemeeter Banana is to set it up and explain along the way. This gives us a more hands on approach which we can test actively. I will cover setting up Voicemeeter in this way: Hardware Outputs, Digital Inputs, then Hardware Inputs. My reason for this is it is hard to test if we have any of the inputs set up if we don't have proper outputs set up. So please install Voicemeeter Banana and the free Virtual Audio Cable that comes with it, you may be required to restart your computer.
+
+Location of Virtual Audio Cable in Voicemeeter after installation:
+'C:\Program Files (x86)\VB\Voicemeeter'
+
+
+## Setting up Voicemeeter Banana
+
+Please open Voicemeeter Banana.
+
+### Saving Settings, Loading Settings, Resetting settings
+Something that typically gets skipped in other Voicemeeter coverage that i've seen is the ability to save settings. At any point you can save your current settings of voicemeeter by going to Menu→Save Settings. This saves a settings file on your computer you can then load up from just a file even on other devices should you want to transfer your settings to another device. If at any point you feel lost and want to restart the setup, you can go to Menu→ Reset Settings (Re-Initialization) and this will make Voicemeeter appear as if you had just installed it, blank with nothing set up.
+
+Image for showing where the menu is
+
+![https://i.imgur.com/bbE1y24.png](https://i.imgur.com/bbE1y24.png)
+
+Image for showing save/load/reset settings
+
+![https://i.imgur.com/bbE1y24.png](https://i.imgur.com/bbE1y24.png)
```

Comment:
Rather than linking out to an external image that may change or be removed, let's import the image into the repository and link to that instead.

Reply:
localized and reformated the images